code/modules/mob/living/simple_animal/abnormality/aleph/titania.dm

```diff
@@ -0,0 +1,157 @@
+//Coded by Kirie Saito! EGO done by Chiemi <3
+/mob/living/simple_animal/hostile/abnormality/titania
+	name = "Titania"
+	desc = "A gargantuan fairy."
+	icon = 'ModularTegustation/Teguicons/32x64.dmi'
+	icon_state = "titania"
+	icon_living = "titania"
+	maxHealth = 3500
+	health = 3500
+	is_flying_animal = TRUE
+	threat_level = ALEPH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(0, 0, 0, 45, 50),
+		ABNORMALITY_WORK_INSIGHT = list(0, 0, 0, 30, 40),
+		ABNORMALITY_WORK_ATTACHMENT = list(0, 0, 10, 20, 35),
+		ABNORMALITY_WORK_REPRESSION = 0
+			)
+	start_qliphoth = 3
+	move_to_delay = 4
+
+	work_damage_amount = 16
+	work_damage_type = RED_DAMAGE
+	can_breach = TRUE
+
+	melee_damage_lower = 92
+	melee_damage_upper = 99		//Will never one shot you.
+	armortype = RED_DAMAGE
+	melee_damage_type = RED_DAMAGE
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.3, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 1)
+	stat_attack = HARD_CRIT
+
+	ego_list = list(
+		/datum/ego_datum/weapon/soulmate,
+		/datum/ego_datum/armor/soulmate
+		)
+//	gift_type = /datum/ego_gifts/soulmate
+
+	var/fairies_spawned = 0
+	var/fairy_spawn_number = 2
+	var/fairy_spawn_time = 4 SECONDS
+	var/fairy_spawn_limit = 100 // Oh boy, what can go wrong?
+	var/list/spawned_mobs = list()
+	var/list/worked = list()
+	var/nemesis			//Who is her nemesis?
+	//The nemesis is referred to as Oberon in the rest of the comments.
+
+//Attacking code
+/mob/living/simple_animal/hostile/abnormality/titania/AttackingTarget()
+	var/mob/living/carbon/human/H = target
+	//Kills the weak immediately.
+	if(get_user_level(H) < 4 && (ishuman(H)))
+		say("I rid you of your pain, mere human.")
+		H.gib()
+		for(var/i=fairy_spawn_number, i>=1, i--)	//This counts down.
+			new /mob/living/simple_animal/hostile/fairyswarm(get_turf(target))
+		return
+
+	if(target == nemesis)	//Deals pale damage to Oberon, fuck you.
+		armortype = PALE_DAMAGE
+		melee_damage_type = PALE_DAMAGE
+		melee_damage_lower = 61
+		melee_damage_upper = 72
+	else if(nemesis)		//If there's still a nemesis, you need to reset the damage
+		armortype = initial(armortype)
+		melee_damage_type = initial(melee_damage_type)
+		melee_damage_lower = initial(melee_damage_lower)
+		melee_damage_upper = initial(melee_damage_upper)
+
+	. = ..()
+
+	if(H.stat == DEAD && target == nemesis)		//Does she slay Oberon personally? If so, get buffed.
+		move_to_delay = 3
+		melee_damage_lower = 110
+		melee_damage_upper = 140
+		adjustBruteLoss(-maxHealth) // Round 2, baby
+
+		to_chat(src, "<span class='userdanger'>[nemesis], my beloved devil, I finally get my revenge.</span>")
+		nemesis = null
+		if(!client)
+			say("Oberon. The abhorrent taker of my child. You are slain.")
+
+//Spawning Fairies
+/mob/living/simple_animal/hostile/abnormality/titania/proc/FairyLoop()
+	//Blurb about how many we have spawned
+	listclearnulls(spawned_mobs)
+	for(var/mob/living/L in spawned_mobs)
+		if(L.stat == DEAD)
+			spawned_mobs -= L
```

Comment:
This should check for Nulls and QDEL-ed mobs for guaranteed removal

Reply:
This is how Egor does it, I can nullcatch but I don't think it would do much